src/slide.jsx

```diff
@@ -33,10 +36,12 @@ const Slide = React.createClass({
   setZoom() {
     const content = React.findDOMNode(this.refs.content);
     const zoom = (content.offsetWidth / config.width);
-    const contentScale = (content.parentNode.offsetHeight / config.height);
+    const contentScaleY = (content.parentNode.offsetHeight / config.height);
+    const contentScaleX = (content.parentNode.offsetWidth / config.width);
```

Comment:
We shouldn't do this. Open up the presentation in regular mode and scale to mobile size, you'll see what I mean.